src/serde_helpers.rs

```diff
@@ -185,6 +191,49 @@ pub mod u64_as_f64 {
     }
 }
 
+/// Contains functions to serialize a [`time::OffsetDateTime`] as a [`crate::DateTime`] and
+/// deserialize a [`time::OffsetDateTime`] from a [`crate::DateTime`].
+///
+/// ```rust
+/// # #[cfg(feature = "chrono-0_4")]
+/// # {
+/// # use serde::{Serialize, Deserialize};
+/// # use bson::serde_helpers::time_offsetdatetime_as_bson_datetime;
+/// #[derive(Serialize, Deserialize)]
+/// struct Event {
+///     #[serde(with = "time_offsetdatetime_as_bson_datetime")]
+///     pub date: time::OffsetDateTime,
+/// }
+/// # }
+/// ```
+#[cfg(feature = "time-0_3")]
+#[cfg_attr(docsrs, doc(cfg(feature = "time-0_3")))]
+pub mod time_offsetdatetime_as_bson_datetime {
```

Comment:
I know this matches the existing pattern, but now that I'm looking at this again, I think we'll want to include the versioning in the names here as well. That way we wont run into clashes if need a `time-0_4` or just `time` feature.

Reply:
Done.